antora-playbook-author.yml

```diff
@@ -20,3 +20,14 @@ asciidoc:
   attributes:
     linkattrs: ''
     toc: ~
+    quarkus_version: '2.9.1.Final'
+    java_min_version: 11
+    maven_min_version: 3.8.1
+    graalvm_min_version: 22.0.0.2
+    spec_version: 0.8
+    kogito_examples_url: https://github.com/kiegroup/kogito-examples.git
+    quarkus_cli_url: https://quarkus.io/guides/cli-tooling
+    spec_website_url: https://serverlessworkflow.io/
+    spec_doc_url: https://github.com/serverlessworkflow/specification/blob/0.8.x/specification.md
+    cloud_events_url: https://cloudevents.io/
+    cloud_events_sdk_url: https://github.com/cloudevents/sdk-java
```

Comment:
I think we are now focusing on one file to have all the variables: serverlessworkflow/antora.yaml.

See: https://github.com/kiegroup/kogito-docs/pull/25/files

Reply:
Done. 
Also updated the examples attribute as follows:

kogito_sw_examples_url: https://github.com/kiegroup/kogito-examples/tree/main/serverless-workflow-examples